src/component/registry/bpmn-model-registry.ts

```diff
@@ -22,12 +22,18 @@ import ShapeBpmnElement, { ShapeBpmnSubProcess } from '../../model/bpmn/internal
 
 export class BpmnModelRegistry {
   private searchableModel: SearchableModel;
+  private loadedSearchableModelCallback: () => void;
 
   computeRenderedModel(bpmnModel: BpmnModel): RenderedModel {
     this.searchableModel = new SearchableModel(bpmnModel);
+    this.loadedSearchableModelCallback?.();
     return toRenderedModel(bpmnModel);
   }
 
+  onSearchableModelLoad(callback: () => void): void {
```

Comment:
❓ rename into `onModelLoad`?

Reply:
I find the name very difficult to understand what it do and what it is for (same for `loadedSearchableModelCallback`.

Reply:
Let's talk about it in person later.
Very often, libraries provide a way to subscribe to "events" with function like `on(event_type, callback_function)`.
Here we only have a single kind of "events" so I proposed to put the kind in the function name directly.

Reply:
Go for `registerOnLoadCallback`

Reply:
done in done in 73db95b